sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -288,3 +288,20 @@ def formula2(n_features, n_features_to_select, step):
                      formula1(n_features, n_features_to_select, step))
         assert_equal(rfecv.grid_scores_.shape[0],
                      formula2(n_features, n_features_to_select, step))
+
+
+def test_rfe_cv_n_jobs():
+    generator = check_random_state(0)
+    iris = load_iris()
+    X = np.c_[iris.data, generator.normal(size=(len(iris.data), 6))]
+    y = iris.target
+
+    rfecv = RFECV(estimator=SVC(kernel='linear'))
+    rfecv.fit(X, y)
+    rfecv_ranking = rfecv.ranking_
+    rfecv_grid_scores = rfecv.grid_scores_
+
+    rfecv.set_params(n_jobs=-1)
```

Comment:
imagine you run this on a machine with 64 CPUs ... don't use -1 but just 2.

although I feel we should not have to test this.